.travis.yml

```diff
@@ -1,67 +0,0 @@
-dist: xenial
```

Comment:
Are we ready to drop the entire file? There are linters there that are waiting for #5786

Reply:
Black, flake, mypy and isort are in pre-commit and migrations and GraphQL schema checks are also in it.

The question for that PR is whether we would want to have those checks in a separate workflow. The issue with pre-commit checks (migrations, GraphQL schema) is that we cannot tell pre-commit to only run them and no the others.